specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2021-07-01/Updates.json

```diff
@@ -234,20 +333,101 @@
       "description": "State of the update.",
       "type": "string",
       "enum": [
+        "Available",
+        "Downloading",
+        "DownloadFailed",
+        "HealthChecking",
+        "HealthCheckFailed",
+        "Invalid",
         "HasPrerequisite",
+        "Recalled",
         "Obsolete",
-        "Ready",
+        "ReadyToInstall",
         "NotApplicableBecauseAnotherUpdateIsInProgress",
-        "Preparing",
         "Installing",
         "Installed",
-        "PreparationFailed",
+        "AttentionNeeded",
         "InstallationFailed"
       ],
       "x-ms-enum": {
         "name": "UpdateState",
         "modelAsString": true
       }
+    },
+    "AvailabilityType": {
+      "description": "Availability type of update.",
+      "type": "string",
+      "enum": [
+        "online",
+        "local",
+        "notify"
+      ],
+      "x-ms-enum": {
+        "name": "AvailabilityType",
+        "modelAsString": true
+      }
+    },
+    "Prerequisite": {
+      "description": "Prerequisite of the update.",
+      "type": "object",
+      "properties": {
+        "title": {
+          "description": "Title of the prerequisite.",
+          "type": "string"
+        },
+        "description": {
+          "description": "Description of the prerequisite.",
+          "type": "string"
+        },
+        "infomationLink": {
+          "description": "Supported link of the prerequisite.",
+          "type": "string"
+        }
+      }
+	},
+    "UpdateStateProperties": {
+      "description": "State related properties of the update.",
+      "type": "object",
+      "properties": {
+        "downloadPercentage": {
+          "description": "Package download percentage.",
+          "type": "integer",
```

Comment:
"progressPercentage" seems fine, because we already have the state which can indicate the current running steps. This just give a overall progress for detailed steps, people need to call to get UpdateRun, so we don't need the progressMessage.